dll/win32/imm32/candidate.c

```diff
@@ -161,20 +161,25 @@ ImmGetCandidateListAW(HIMC hIMC, DWORD dwIndex, LPCANDIDATELIST lpCandList, DWOR
     LPCANDIDATELIST pCL;
 
     pClientImc = ImmLockClientImc(hIMC);
-    if (IS_NULL_UNEXPECTEDLY(pClientImc))
```

Comment:
Note that if you were to keep this macro in a header of imm32, wine wouldn't have any problem with this.